docs/concepts/linq.rst

```diff
@@ -0,0 +1,268 @@
+LINQ
+====
+
+What is it?
+-----------
+
+LINQ provides language-level querying capabilities and `higher-order function <https://en.wikipedia.org/wiki/Higher-order_function>`_ API to C# and VB (note that F# already has this capability by nature of being a functional programming language).  It is used either via native query syntax,
+
+.. code-block:: c#
+
+	var dotNetProgrammers = from p in programmers
```

Comment:
This example is a little confusing to me. It suggests that programmers that are new to LINQ become Linq experts by some magic and that this forms a set called .NET programmers. Thoughts?